src/test/java/teammates/test/cases/webapi/UpdateFeedbackResponseActionTest.java

```diff
@@ -25,6 +27,16 @@
  */
 public class UpdateFeedbackResponseActionTest extends BaseActionTest<UpdateFeedbackResponseAction> {
 
+    private StudentAttributes student1InCourse1;
+    private StudentAttributes student2InCourse1;
+    private StudentAttributes student5InCourse1;
+    private InstructorAttributes instructor2OfCourse1;
+    private InstructorAttributes instructor1OfCourse1;
+    private FeedbackResponseAttributes student1ResponseToStudent1;
+    private FeedbackResponseAttributes testModerateResponse;
```

Comment:
This is only utilized in `testAccessControl_giverNotModeratedStudent_inaccessible`